cifar10-multiannotator/utils/model_training.py

```diff
@@ -0,0 +1,272 @@
+import sys
+
+sys.path.insert(0, "../")
+
+import numpy as np
+import pandas as pd
+import pickle
+import datetime
+import os
+from pathlib import Path
+from typing import Dict, Tuple
+
+import cleanlab
+from autogluon.vision import ImagePredictor, ImageDataset
```

Comment:
Add specific version of autogluon to the dependencies of this **examples/** repo?

https://github.com/cleanlab/examples/blob/master/requirements.txt